### ISSUE TITLE
 UI of the footer can be improved

### ISSUE THREAD

**Issue by Prajnaprabhu3** [hash-define-organization/website-update#32]
I can work on it

**Comment by WINOFFRG** [comment]
Yes definitely it can be! Please go ahead and start working on it,

**Comment by Sandhya312** [comment]
I am a GSSOC'22 contributor and I want to work on it. Please assign it to me.

**Comment by WINOFFRG** [comment]
Sure @Sandhya312 , Please go ahead. You can start working on it,

**Comment by Chitrakshi2531** [comment]
@WINOFFRG  If this issue is still open I would like to work on it

**Comment by WINOFFRG** [comment]
Yes @Chitrakshi2531 ! I have assigned the issue to you. You can start working on it. Good Luck!
Let me know if you encounter any issue with the project
